convex/cortex/projects/TestProject/Makefile

```diff
@@ -152,7 +156,8 @@ INCDIR = $(PORTINC) $(KERNINC) \
 MCU  = cortex-m3
 
 #TRGT = arm-elf-
-TRGT = arm-none-eabi-
+#TRGT = arm-none-eabi-
+TRGT = $(ARM_TOOLS_FOLDER)/arm-none-eabi-

```

Comment:
You might consider taking out the `/`. if the variable isn't set, then it will go to the root directory.